app/assets/javascripts/application.js

```diff
@@ -42,42 +42,30 @@ function show_notification_blood(msg){
 }
 
 
-function prepare_to_initialize(q) {
+function prepare_to_initialize() {
   $(document).ready(function() {
-   google.maps.event.addDomListener(window, 'load',function () { initialize(q);});
-});
+   google.maps.event.addDomListener(window, 'load', initialize());
+ });
 }
 
 
- function initialize(x) {
+ function initialize() {
 
-  var input = document.getElementById(x);
+  var input = document.getElementsByClassName('search');
   var options = {
     componentRestrictions: {country: "eg"}
   };
+ var autocomplete = new google.maps.places.Autocomplete(input[0], options);
 
-  var autocomplete = new google.maps.places.Autocomplete(input,options);
-
- if (x == "searchTextField") {
-       google.maps.event.addListener(autocomplete, 'place_changed', function () {
+        google.maps.event.addListener(autocomplete, 'place_changed', function () {
         var place = autocomplete.getPlace();
-        document.getElementById('addName').value = document.getElementById('searchTextField').value;
-        document.getElementById('addLat').value = place.geometry.location.lat();
-        document.getElementById('addLng').value = place.geometry.location.lng();
-   });
-  }
-
-else {
-    google.maps.event.addListener(autocomplete, 'place_changed', function () {
-     var place = autocomplete.getPlace();
-     document.getElementById('hospital_addName').value = document.getElementById('hospital_searchTextField').value;
-     document.getElementById('hospital_addLat').value = place.geometry.location.lat();
-     document.getElementById('hospital_addLng').value = place.geometry.location.lng();
-   });
-  }
+        document.getElementsByClassName('add_name')[0].value = document.getElementsByClassName('search')[0].value;
+        document.getElementsByClassName('add_lat')[0].value = place.geometry.location.lat();
+        document.getElementsByClassName('add_lng')[0].value = place.geometry.location.lng();
+ });
```

Comment:
Indentation please, you can install http://editorconfig.org/ and add this shortcut to sublime if you are using it https://coderwall.com/p/7yxpdw/auto-indenting-on-sublime-text-3